tests/playbooks/tests_switch_provider.yml

```diff
@@ -0,0 +1,66 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# This file was generated by ensure_provider_tests.py
+---
+# set network provider and gather facts
+- hosts: all
+  name: Switch initscripts provider to nm
+  vars:
+    interface: LSR-TST-br34
+  tasks:
+    - name: set fact to use initscripts network_provider
+      set_fact:
+        network_provider: initscripts
+      tags:
+        - always
+    - name: "Create test bridge {{ interface }} via initscripts provider"
+      include_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          - name: "{{ interface }}"
+            state: up
+            type: bridge
+            ip:
+              dhcp4: false
+              auto6: false
+    - include_tasks: tasks/assert_device_present.yml
+    - name: "Take the profile {{ interface }} down via initscripts provider"
+      include_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          - name: "{{ interface }}"
+            persistent_state: absent
+            state: down
+            type: bridge
+    # The initscripts should not remove the interface for down/absent
+    - include_tasks: tasks/assert_device_present.yml
+    - name: set fact to use nm network_provider
+      set_fact:
+        network_provider: nm
+      tags:
+        - always
+    - name: "Create test bridge {{ interface }} via nm provider"
+      include_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          - name: "{{ interface }}"
+            state: up
+            type: bridge
+            ip:
+              dhcp4: false
+              auto6: false
+    - include_tasks: tasks/assert_device_present.yml
+    - name: "Remove bridge {{ interface }} via nm provider"
+      include_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
```

Comment:
how about importing `tasks/remove+down_profile.yml`  here?

Reply:
Please, notice the comment: `# NetworkManager should not remove pre-exist interface for down/absent`. So this is why we are asserting that the device is present and then deleting it with `delete_interface.yml` instead of doing it in one step.

Thanks!

Reply:
Oh, my bad. I didn't read this correctly. Yes, let's use remove+down_profile.yml, thanks!